db/seeds.rb

```diff
@@ -1,50 +1,57 @@
+# frozen_string_literal: true
+
 require 'csv'
 require 'faker'
 
 # add cities in DB
 
-City.all.destroy_all
+City.find_each(&:destroy!)
 
-source_file = "/Users/eleonore/Documents/Code/enm-city-allocator/db/enm_cities_seats.csv"
+source_file = '/Users/eleonore/Documents/Code/enm-city-allocator/db/jurisdictions_and_choices.csv'
 
 CSV.foreach(source_file, headers: true) do |row|
-	City.create(
-		name: row[1],
-		nb_seats: row[2]
-		)
+  City.create!(
+    name: row[0],
+    nb_seats: row[1],
+    wishes: [row[2], row[3], row[4], row[5], row[6], row[7]].map(&:to_i)
+  )
 end
 
 # add students in DB
 
-Student.all.destroy_all
+Student.find_each(&:destroy!)
 
-Student.create(name: "Anna Thuleau")
+Student.create!(name: 'Anna Thuleau')
 
-335.times {
-	Student.create!(
-	name: Faker::Name.unique.name
-	)
-}
+335.times do
+  Student.create!(
+    name: Faker::Name.unique.name
+  )
+end
 
 # add choices in DB
 
-Choice.all.destroy_all
-
-Student.all.each do |student|
-	rank = 1
-	6.times {
-		n = rand(100)
-		if n < 20
-			random_city = City.find_by name: 'PARIS (PI)'
-		elsif n < 40
-			random_city = City.find_by name: 'BORDEAUX (PI)'
-		else
-			random_city = City.all.sample(100)[50]
-		end
-		Choice.create!(
-			city: random_city, 
-			student: student,
-			rank: rank)
-		rank += 1
-	}
-end
\ No newline at end of file
+Choice.find_each(&:destroy!)
+
+City.order("nb_seats DESC").all.each do |city|
+  city.wishes.each_with_index do |wish, ind|
```

Comment:
Naming: `index`. It never hurts to be explicit

Reply:
Thanks. I thought that was a common naming and that when possible (and when the naming is explicit enough) it was a best practice to have variables with short names, but dully noted.